### ISSUE TITLE
How to delete my SSMAgent1 (through 4) and start over?

### ISSUE THREAD

**Issue by Dogmatic** [mrhid6/SatisfactoryServerManager#129]
I had a working server... then I tried to put a mod on... things didn't work after that.

I tried creating new servers and trying again, nothing worked.  Just want to go back to the basics, but can't get anything to load at this point.  

Want a fresh start.
Thanks!

**Comment by mrhid6** [comment]
Make sure you download the save file so nothing gets lost first before deleting

**Comment by Dogmatic** [comment]
Thanks, while that command (docker remove -f SSMAgent1) didn't work for me (I had already done enough to mess it up I'm sure).
It did point me to the fact I could just uninstall docker and that should do the trick.  

It did, all the SSM Agents were removed.  But the SSM application will not now allow me to create a server.  Just has a little pop-up saying it failed.

I ran the install for SSM again to make sure docker was installed by it, which it was, but then it stopped saying SSM was already installed.  Still won't let me create a server.
